src/server/game/Server/Protocol/Opcodes.h

```diff
@@ -24,11 +24,18 @@
 #define _OPCODES_H
 
 #include "Common.h"
+#include <iomanip>
 
-/// List of Opcodes
-enum Opcodes
+enum OpcodeMisc : uint16
+{
+    MAX_OPCODE                                        = 0x3FFF,
+    NUM_OPCODE_HANDLERS                               = (MAX_OPCODE + 1),
+    UNKNOWN_OPCODE                                    = (0x7FFF + 1),
```

Comment:
to fit UNKNOWN_OPCODE value into uint16 I changed the value from 0xFFFF + 1 to 0x7FFF + 1. I don't see a reason for this to cause any issue.